assets/grafana/celeborn-dashboard.json

```diff
@@ -1472,7 +1472,7 @@
             "type": "prometheus",
             "uid": "${DS_PROMETHEUS}"
           },
-          "description": "The count of shuffle fallbacks.",
+          "description": "The total count of shuffle.",
```

Comment:
Seems the `shuffle` count including both celeborn shuffle and spark built-in shuffle, we need to enhance the description to make it more clear